app/controllers/api/v2/hosts_controller.rb

```diff
@@ -238,6 +238,18 @@ def rebuild_config
         end
       end
 
+      api :GET, "/hosts/:id/template/:kind", N_("Preview rendered provisioning template content")
+      param :id, :identifier_dottable, :required => true
+      param :kind, String, :required => true, :desc => N_("Template kinds, available values: %{template_kinds}") % { :template_kinds => TemplateKind.all.map(&:name).join(", ") }
```

Comment:
This is getting evaluated when this controller is loaded, but the TempateKind table may not be available (e.g. before db:migrate).  It needs to be lazier.